### PR TITLE
feat: enableUnfinishedFeatures flag and hidden preferences button

### PR DESCRIPTION
**Pull request by gorjan5sk**
- Implemented enableUnfinishedFeatures flag, applied as env var and cascades to AppState where it can be read
- Implemented rudimentary Preferences button to show/hide preferences screen, currently it fails silently since the preferences screen is in another PR

### CODE REVIEW

File: app/assets/javascripts/ui_models/app_state/app_state.ts [hunk 1]
```diff
@@ -80,6 +80,7 @@ export class AppState {
     application: WebApplication,
     private bridge: Bridge
   ) {
+    console.log('enableUnfinishedFeatures', this.enableUnfinishedFeatures);
```
Comment (mobitar) [inline app/assets/javascripts/ui_models/app_state/app_state.ts:83]:
```suggestion
```

Comment (gorjan5sk) [reply app/assets/javascripts/ui_models/app_state/app_state.ts]: Fixed.

Comment (antsgar) [reply app/assets/javascripts/ui_models/app_state/app_state.ts]: Should we maybe only print this log on the dev app?

**Review by antsgar — APPROVED** [review]

File: app/assets/javascripts/ui_models/app_state/app_state.ts (hunk 1 above)
Comment (gorjan5sk) [reply app/assets/javascripts/ui_models/app_state/app_state.ts]: I removed that, we don't need it. :D I'm gonna resolve this conversation.

**Pull request merged**